<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeakChargingInformationView</class>
 <widget class="QWidget" name="PeakChargingInformationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="6,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="font">
        <font>
         <pointsize>23</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: \*url();

QPushButton {font: 12pt ; } </string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <widget class="QWidget" name="tabTip">
        <property name="styleSheet">
         <string notr="true">border-image: \*url();</string>
        </property>
        <attribute name="title">
         <string>尖</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <widget class="TableWidgetKeyboard" name="tableWidgetTip">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 170, 255);</string>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>83</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>40</number>
           </attribute>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <column>
            <property name="text">
             <string>开始小时</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>开始分钟</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>结束小时</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>结束分钟</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>SOC限值%</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>电流限值/A</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabPeak">
        <attribute name="title">
         <string>峰</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <widget class="TableWidgetKeyboard" name="tableWidgetPeak">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 170, 255);</string>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>83</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>40</number>
           </attribute>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <column>
            <property name="text">
             <string>开始小时</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>开始分钟</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>结束小时</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>结束分钟</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>SOC限值%</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>电流限值/A</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabFlat">
        <attribute name="title">
         <string>平</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <widget class="TableWidgetKeyboard" name="tableWidgetFlat">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 170, 255);</string>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>83</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>40</number>
           </attribute>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <column>
            <property name="text">
             <string>开始小时</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>开始分钟</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>结束小时</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>结束分钟</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>SOC限值%</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>电流限值/A</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabValley">
        <attribute name="title">
         <string>谷 </string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <widget class="TableWidgetKeyboard" name="tableWidgetValley">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 170, 255);</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>83</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>40</number>
           </attribute>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <column>
            <property name="text">
             <string>开始小时</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>开始分钟</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>结束小时</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>结束分钟</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>SOC限值%</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>电流限值/A</string>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="background">
             <color>
              <red>0</red>
              <green>170</green>
              <blue>255</blue>
             </color>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3,1,1">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>68</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="labelPeakChargingOpenOrClose">
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>错峰充电开启关闭</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxPeakChargingOpenOrClose">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 170, 255);</string>
           </property>
           <item>
            <property name="text">
             <string>关闭</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>开启</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>68</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSave">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton { background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); } 
QPushButton:hover:pressed{ background-color: rgb(0,0,127); color:rgb(255, 255, 255);} 
QPushButton:hover:!pressed{ background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); }
QPushButton {font: 18pt ; } </string>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableWidgetKeyboard</class>
   <extends>QTableWidget</extends>
   <header>TableWidgetKeyboard.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
